system/dtb/xenvm-4.2.dtb.license

```diff
@@ -0,0 +1 @@
+SPDX-License-Identifier: GPL-2.0
```

Comment:
As the CI check complains, it's necessary to include the copyright info.

Also the [~`GPL-2.0`~](https://spdx.org/licenses/GPL-2.0.html) SPDX identifier is deprecated - [`GPL-2.0-only`](https://spdx.org/licenses/GPL-2.0-only.html) or [`GPL-2.0-or-later`](https://spdx.org/licenses/GPL-2.0-or-later.html) should be used instead. The fact that the original `xenvm-4.2.dts` file contains it is their business :-)

Linus Torvalds explicitly writes [in the COPYING file](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/COPYING?id=b24413180f56#n8) that

>  (...) the only valid version of the GPL as far as the kernel
>  is concerned is _this_ particular version of the license (ie v2, not
>  v2.2 or v3.x or whatever), unless explicitly otherwise stated.

so here it is clear.

Also missing licenses that have not been used before (i.e. which are not in the [`LICENSES/`](https://github.com/kaitai-io/kaitai_struct_samples/tree/c3b8656c458b70272d1976b8aa9609eb99967e75/LICENSES) folder yet) need to be downloaded to `LICENSES/` - the command `reuse download --all` will resolve and download all missing licenses (and tells you what license identifiers are deprecated or invalid, if any). But I can do that in an additional commit in each PR where it occurs, no big deal.